.gitignore

```diff
@@ -21,3 +21,4 @@
 /tmp
 public/system # For paperclip attachments in development and test environments
 *~
+.*.swp
```

Comment:
@betesh This is no problem, but I recommend you ignore this globally.

~/.gitignore

    .*.swp

~/.gitconfig

    [core]
        excludesfile = /Users/nilbus/.gitignore